src/lib/cli/apiConfig.js

```diff
@@ -64,6 +63,11 @@ export async function checkIsVIP(): Promise<boolean> {
 	return !! isVip;
 }
 
+export async function checkIfUserIsVip() {
+	const res = await featureFlags.get();
```

Comment:
Is this cached? If not, we'll be adding a lot of requests to every event tracking call which will probably slow things down a fair bit.

Reply:
Good point! What do you think should be a reasonable TTL for this? 1 day? or maybe even more?

Reply:
@mjangda , you did mean caching on Parker, right?

Reply:
I meant in the CLI.

If I do:

```
await checkIfUserIsVip();
await checkIfUserIsVip();
await checkIfUserIsVip();
await checkIfUserIsVip();
await checkIfUserIsVip();
```

will that make 5 HTTP requests?

Reply:
Looks like the `featureFlags.get()` makes an API request to Parker via the Apollo Client which we have set up to use an in-memory cache (https://github.com/Automattic/vip/blob/4955be2eaae70466e597b3112fb557dcd92a8556/src/lib/api.js#L70), so I think we should be okay.

Reply:
Or, maybe not :)

Without this PR (one call to Parker):

```
❯ DEBUG=agent-base ./dist/bin/vip-app.js 309 > /dev/null
  agent-base Resolving socket for 'https:' request: 'POST /graphql' +0ms
```

With this PR (3 extra calls to Parker, with an added time of ~600ms):

```
❯ DEBUG=agent-base ./dist/bin/vip-app.js 309 > /dev/null
  agent-base Resolving socket for 'https:' request: 'POST /graphql' +0ms
  agent-base Resolving socket for 'https:' request: 'POST /graphql' +336ms
  agent-base Resolving socket for 'https:' request: 'POST /graphql' +158ms
  agent-base Resolving socket for 'https:' request: 'POST /graphql' +295ms
```

Reply:
We probably need a basic cache within the feature flags lib. Can just store in memory without the need to persist across CLI commands (i.e. just store the feature flags data in a variable in the module).

Reply:
Thanks @mjangda !! Got sidetracked by another higher priority PR. Will get back on this soon!

Reply:
So the Apollo Client's in memory cache does work. But because of the way we use the Client library in the `feature-flag.js` file (and most other files TBH) - by instantiating a new object of the Parker API library every time we make an API call, we cannot leverage the cached objects. 

Please see https://github.com/Automattic/vip-go-admin-console/blob/c672d42633c53e17baef7ced9e79b8bb0e9b1412/components/spinner/index.jsx

Reply:
Nice fix! Thanks!